### PR TITLE
Replaced npm with yarn

### PR DESCRIPTION
**Pull request by ignaciosantise**
## Summary
Removed npm as a dependency, using yarn instead

Related to issue https://github.com/Wolox/wolmo-bootstrap-react-native/issues/61

## Trello Card
https://trello.com/c/mLu4ijOG/173-eliminar-npm

### CODE REVIEW

File: generators/app/tasks/appSetup/packageJsonScripts.js [hunk 1]
```diff
@@ -5,9 +5,9 @@ module.exports = function packgeJsonScripts() {
   packageJson.scripts.ios = 'react-native run-ios --scheme qa';
   packageJson.scripts.test = 'jest';
   packageJson.scripts.coverage = 'jest --coverage';
-  packageJson.scripts.clean = 'rm -rf $TMPDIR/react-* && watchman watch-del-all && npm cache clean';
+  packageJson.scripts.clean = 'rm -rf $TMPDIR/react-* && watchman watch-del-all && yarn cache clean';
   packageJson.scripts['force-clean'] =
-    'yarn run android:clean && rm -rf $TMPDIR/react-* && watchman watch-del-all && rm -rf ios/build && rm -rf ios/Pods && rm -rf node_modules/ && npm cache clean && yarn install && cd ios/ && pod install';
+    'yarn run android:clean && rm -rf $TMPDIR/react-* && watchman watch-del-all && rm -rf ios/build && rm -rf ios/Pods && rm -rf node_modules/ && yarn cache clean && yarn install && cd ios/ && pod install';
```
Comment (sfernandez11) [inline generators/app/tasks/appSetup/packageJsonScripts.js:10]: You could optimize this line using the `clean` script: `yarn run android:clean && yarn run clean && rm -rf ios/build && rm -rf ios/Pods && rm -rf node_modules/ && yarn install && cd ios/ && pod install"`

File: generators/app/templates/.woloxci/Dockerfile [hunk 2]
```diff
@@ -4,7 +4,7 @@ WORKDIR /home/node
 
 COPY package.json .
 
-RUN npm install
+RUN yarn install
```
Comment (felire) [inline generators/app/templates/.woloxci/Dockerfile:7]: If we are going to use bitrise for CI in all of our projects, Don't we have to delete the `.woloxci` from this folder?

Comment (ignaciosantise) [reply generators/app/templates/.woloxci/Dockerfile:7]: yep, but thats another PR

Comment (felire) [reply generators/app/templates/.woloxci/Dockerfile:7]: oka

**Pull request merged**